services/web/server/src/simcore_service_webserver/functions/_functions_service.py

```diff
@@ -457,15 +458,41 @@ async def get_function_user_permissions(
     )
 
 
+async def get_function_group_permissions(
+    app: web.Application,
+    *,
+    user_id: UserID,
+    product_name: ProductName,
+    function_id: FunctionID,
+) -> list[FunctionGroupAccessRights]:
+    access_rights_list = await _functions_repository.get_group_permissions(
+        app=app,
+        user_id=user_id,
+        product_name=product_name,
+        object_ids=[function_id],
+        object_type="function",
+    )
+
+    for object_id, access_rights in access_rights_list:
```

Comment:
is there not a problem here?
why do you nned to check the object_id? ist he function `get_group_permissions` not already filtering by `function_id`?

Reply:
It's a batch call, it accepts a list of objects